src/pages/index.tsx

```diff
@@ -8,308 +9,72 @@ const Home: NextPage = () => {
-          Yet tell me not, for I have heard it all.
-          Here's much to do with hate, but more with love.
-          Why, then, O brawling love! O loving hate!
-          O any thing, of nothing first create!
-          O heavy lightness! serious vanity!
-          Mis-shapen chaos of well-seeming forms!
-          Feather of lead, bright smoke, cold fire,
-          sick health!
-          Still-waking sleep, that is not what it is!
-          This love feel I, that feel no love in this.
-          Dost thou not laugh?
-          BENVOLIO
-          No, coz, I rather weep.
-          ROMEO
-          Good heart, at what?
-          BENVOLIO
+        <div className="flex flex-col w-full px-28 mt-10 text-green-800">
+          <div className="text-5xl font-light">
+            Welcome to EMURR
+          </div>
+          <div className="flex justify-between w-full mt-5">
+            <div className="w-3/5 text-xl font-light text-justify">
+              EMURR? I don’t even know her! Ok, ok. I’ll put some lorem ipsum here or whatever. Cupcake ipsum dolor sit amet muffin donut bonbon cake. Cheesecake tiramisu dragée cotton candy wafer icing gingerbread. Jujubes jelly I love ice cream I love croissant pudding. Dragée I love jelly beans I love macaroon marzipan candy dragée.
+            </div>
+            <div className="flex justify-center w-2/5 items-center">
+              <Logo />
+            </div>
+          </div>
+          <div className="text-5xl mt-40 font-light">
+            Meet the team
+          </div>
+          <div className="flex flex-col justify-between w-full mt-5">
+            <div className="text-2xl text-center font-light">
+              Royal Court of House CHDR
+            </div>
+            <div className="flex justify-between my-10">
+              <div>
+                <img src="/images/logo_vert_5.png" className="w-40 h-40" />
+                <div className="text-center font-semibold">EMURR EMURR</div>
+              </div>
+              <div>
+                <img src="/images/logo_vert_5.png" className="w-40 h-40" />
+                <div className="text-center font-semibold">EMURR EMURR</div>
+              </div>
+              <div>
+                <img src="/images/logo_vert_5.png" className="w-40 h-40" />
+                <div className="text-center font-semibold">EMURR EMURR</div>
+              </div>
+              <div>
+                <img src="/images/logo_vert_5.png" className="w-40 h-40" />
+                <div className="text-center font-semibold">EMURR EMURR</div>
+              </div>
+            </div>
+            <div className="text-2xl text-center font-light">
+              Knights of House CHDR
+            </div>
+            <div className="flex justify-center my-10">
+              <div>
+                <img src="/images/logo_vert_5.png" className="w-40 h-40 mr-20" />
+                <div className="text-center font-semibold">EMURR EMURR</div>
+              </div>
```

Comment:
the images and text arent aligned and the margins aren't a pleasant way to manage space between same/similar elements. instead i would use `gap` like so:
```
            <div className="flex justify-center my-10 gap-3">
              <div>
                <img src="/images/logo_vert_5.png" className="w-40 h-40" />
                <div className="text-center font-semibold">EMURR EMURR</div>
              </div>
```

you might notice your text is still off so we can make the div holding the image and the text a flex box and align its items, otherwise `text-center` will become offset from the image box. text-center with flex centering is redundant so i remove it:
```
            <div className="flex justify-center my-10 gap-3">
              <div className="flex justify-center items-center">
                <img src="/images/logo_vert_5.png" className="w-40 h-40" />
                <div className="font-semibold">EMURR EMURR</div>
              </div>
```

Reply:
i realize maybe a bigger gap is necessary. perhaps `gap-10`?